docker_images/flair/app/pipelines/token_classification.py

```diff
@@ -41,15 +41,15 @@ def __call__(self, inputs: str) -> List[Dict[str, Any]]:
                     "score": current_data_point.score,
                 }
                 entities.append(current_entity)
-            elif isinstance(label, Span):
-                if len(label.tokens) == 0:
+            elif isinstance(current_data_point, Span):
```

Comment:
Do our tests cover both use cases?

Reply:
Yeah, locally I added some debug outputs after lines 35 and  45. Our tests use `flair/chunk-english-fast` and `flair/upos-english-fast`:

Output is:

```
docker_images/flair/tests/test_api_token_classification.py::TokenClassificationTestCase_0_flair_chunk_english_fast::test_simple NER model!

docker_images/flair/tests/test_api_token_classification.py::TokenClassificationTestCase_1_flair_upos_english_fast::test_simple PoS Tagging model!
```

So both cases are tested :)